app/controllers/application_controller.rb

```diff
@@ -2,11 +2,13 @@ class ApplicationController < ActionController::API
   before_action :set_current_user
 
   def issue_token(payload)
-    JWT.encode(payload, ENV["RAILS_SECRET"])
+    # JWT.encode(payload, ENV["RAILS_SECRET"])
+    JWT.encode(payload, "foobar")
```

Comment:
TODO: hard-coded this to "foobar" to get it working. Need to find out how to set up the RAILS_SECRET env var properly in Rails.